docs/en/operations/settings/settings.md

```diff
@@ -1262,4 +1262,23 @@ Possible values:
 
 Default value: 16.
 
+## always_fetch_merged_part {#always_fetch_merged_part}
+
+Prohibits data parts merging at replicas in [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md)-engine tables.
```

Comment:
Let's change `MergeTree` for `ReplicatedMergeTree*` and give a link to https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/replication/ (relative link).